lib/Mojo/Server/Prefork.pm

```diff
@@ -182,6 +182,7 @@ sub _wait {
     next unless my $w = $self->{pool}{$1};
     @$w{qw(healthy time)} = (1, $time) and $self->emit(heartbeat => $1);
     $w->{graceful} ||= $time if $2;
+    $w->{quit}++ if $2;
```

Comment:
That's two redundant `if $2` checks.

Reply:
Ah, yeah, was attempting to match the existing style. I'll put them both in the same block